### PR TITLE
Feature/Init_project_and_get_limit_message

### PR DESCRIPTION
**Pull request by Haiss2**
Please follow [Binance FIX API docs ](https://developers.binance.com/docs/binance-spot-api-docs/fix-api)

### CODE REVIEW

File: client.go [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+}
+
+func (c *Client) Start() error {
+	if err := c.initiator.Start(); err != nil {
+		c.l.Errorw("Failed to initialize initiator", "error", err)
+		return err
+	}
+
+	// Wait for the session to be authorized by the server.
+	for !c.IsConnected() {
```
Comment (hiepnv90) [inline client.go]: Should add timeout for it in case there is something bad happens.

File: client.go [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+	err = client.Start()
+	if err != nil {
+		client.l.Errorw("Failed to start fix connection", "error", err)
+		return nil, err
+	}
+
+	return client, nil
+}
+
+func (c *Client) Start() error {
```
Comment (hiepnv90) [inline client.go]:
```suggestion
func (c *Client) Start(ctx context.Context) error {
```

File: client.go (hunk 1 above)
Comment (Haiss2) [reply client.go]: Yeah, I updated at [d7c284d](https://github.com/KyberNetwork/binance_fix_api/pull/1/commits/d7c284d8b95029ff24dd7a3ffdaf51b761f8926a)

**Review by hiepnv90 — APPROVED** [review]
LGTM! :+1:

**Pull request merged**